IdentitySdkCore/IdentitySdkCore/Classes/ReachFive.swift

```diff
@@ -116,20 +116,20 @@ public class ReachFive: NSObject {
                         _ = completion(authToken)
                     }
                 }
-        }.onFailure { error in
-            var messageAlert = ""
-            switch error {
-            case .RequestError(let requestErrors):
-                messageAlert = requestErrors.errorUserMsg!
-            case .TechnicalError(let technicalError):
-                messageAlert = (technicalError.apiError?.errorUserMsg)! as String
-            default:
-                messageAlert = error.localizedDescription
+            }.onFailure { error in
+                var messageAlert = ""
+                switch error {
+                case .RequestError(let requestErrors):
+                    messageAlert = requestErrors.errorUserMsg!
+                case .TechnicalError(let technicalError):
+                    messageAlert = (technicalError.apiError?.errorUserMsg)! as String
+                default:
+                    messageAlert = error.localizedDescription
+                }
+                let alert = UIAlertController(title: "Error", message:messageAlert, preferredStyle: UIAlertController.Style.alert)
+                alert.addAction(UIAlertAction(title: "Dismiss", style: UIAlertAction.Style.default, handler: nil))
+                viewController.present(alert, animated: true, completion: nil)
```

Comment:
This is a library, choosing how to present an error to a user is up to our clients who are using the library.

Reply:
This PR is based on Master, i have already fix it in the previous PR (Error Handling)